### DISCUSSION TITLE
Passing email to federated idp does not work when using SAML (Pass Subject)

### DISCUSSION THREAD

**Discussion by gerardnorton** [sventorben/keycloak-home-idp-discovery#474] [General]
![Captura desde 2024-11-26 14-03-18](https://github.com/user-attachments/assets/bba6ff63-72fa-4b50-a785-cc124970b8e2)

![Captura desde 2024-11-26 14-32-51](https://github.com/user-attachments/assets/90fc8a9a-795d-446c-9883-74ba145c71cb)

According to the [official documentation](https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/24.0/html/server_administration_guide/identity_broker#identity_broker_saml_login_hint), we can use the ‘Pass Subject’ option to add the ‘login_hint’ parameter in the request to the federated idp when using SAML integration. I'm testing this with Okta and I see that using the extension doesn't work.

![Captura desde 2024-11-26 14-31-34](https://github.com/user-attachments/assets/98f6d57b-6c31-4777-b002-0cba22c3c899)

**Comment by sventorben** [comment]
Hey @gerardnorton,
which version of this plugin and Keycloak do you use?
